<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>535</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Texture Scaler</string>
  </property>
  <property name="frameShape" stdset="0">
   <string>NoFrame</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QFrame" name="frame_2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>9</number>
             </property>
             <property name="topMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QPushButton" name="restart_btn">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Restart</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/icon_ant/icons/ant-svg/reload 30.svg</normaloff>:/icon_ant/icons/ant-svg/reload 30.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="path_label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <bold>true</bold>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string>Choose Directory or Textures from Disk</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="open_path_btn">
               <property name="minimumSize">
                <size>
                 <width>84</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Open Folder</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/icon_ant/icons/ant-svg/folder-open-fill 363.svg</normaloff>:/icon_ant/icons/ant-svg/folder-open-fill 363.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="open_file_btn">
               <property name="minimumSize">
                <size>
                 <width>84</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Open Images</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/icon_ant/icons/ant-svg/file-copy-fill 357.svg</normaloff>:/icon_ant/icons/ant-svg/file-copy-fill 357.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>550</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>1000</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">selection-background-color: rgb(204, 232, 255);</string>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="showDropIndicator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="dragEnabled">
                <bool>false</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="verticalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="horizontalScrollMode">
                <enum>QAbstractItemView::ScrollPerPixel</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DashLine</enum>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="font">
                 <font/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>√</string>
                </property>
                <property name="font">
                 <font/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Original</string>
                </property>
                <property name="font">
                 <font/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
                <property name="background">
                 <color alpha="120">
                  <red>116</red>
                  <green>151</green>
                  <blue>156</blue>
                 </color>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Scaled</string>
                </property>
                <property name="font">
                 <font/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Tiled</string>
                </property>
                <property name="font">
                 <font/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>Select</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QFrame" name="frame_9">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout">
                      <item>
                       <widget class="QPushButton" name="select_all_btn">
                        <property name="text">
                         <string>All</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="select_none_btn">
                        <property name="text">
                         <string>None</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_7">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_5">
                      <item>
                       <widget class="QLineEdit" name="size_thres_entry">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>40</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>50</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>9</pointsize>
                          <stylestrategy>PreferAntialias</stylestrategy>
                         </font>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="maxLength">
                         <number>5</number>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                        <property name="placeholderText">
                         <string>2048</string>
                        </property>
                        <property name="clearButtonEnabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="set_thres_btn">
                        <property name="text">
                         <string>By Threshold</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QGroupBox" name="set_res_group">
                  <property name="title">
                   <string>Set Resolution</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <layout class="QGridLayout" name="gridLayout">
                     <item row="1" column="0">
                      <widget class="QPushButton" name="res_512_btn">
                       <property name="text">
                        <string>512</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QPushButton" name="res_1024_btn">
                       <property name="text">
                        <string>1024</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QPushButton" name="res_quater_btn">
                       <property name="text">
                        <string>Quater</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QPushButton" name="res_2048_btn">
                       <property name="text">
                        <string>2048</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QPushButton" name="res_4096_btn">
                       <property name="text">
                        <string>4096</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QPushButton" name="res_half_btn">
                       <property name="text">
                        <string>Half</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QPushButton" name="res_128_btn">
                       <property name="text">
                        <string>128</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QPushButton" name="res_256_btn">
                       <property name="text">
                        <string>256</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_8">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_6">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QLineEdit" name="size_res_entry">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>40</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>50</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="font">
                         <font>
                          <pointsize>9</pointsize>
                          <stylestrategy>PreferAntialias</stylestrategy>
                         </font>
                        </property>
                        <property name="maxLength">
                         <number>5</number>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                        <property name="placeholderText">
                         <string>1024</string>
                        </property>
                        <property name="clearButtonEnabled">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="set_res_btn">
                        <property name="text">
                         <string>Set Resolution</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>0</height>
                </size>
               </property>
               <property name="value">
                <number>60</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textVisible">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame_10">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QRadioButton" name="overwrite_btn">
                  <property name="text">
                   <string>Overwrite Existing Texture</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rename_old_btn">
                  <property name="text">
                   <string>Rename Existing Texture</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="execute_btn">
                  <property name="minimumSize">
                   <size>
                    <width>220</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <stylestrategy>PreferAntialias</stylestrategy>
                   </font>
                  </property>
                  <property name="text">
                   <string>Execute</string>
                  </property>
                  <property name="icon">
                   <iconset resource="icons.qrc">
                    <normaloff>:/icon_ant/icons/ant-svg/shrink 213.svg</normaloff>:/icon_ant/icons/ant-svg/shrink 213.svg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>60</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
